readalongs/text/util.py

```diff
@@ -1,368 +1,374 @@
-            elif unit_part == "m":
-                time_in_ms += int(numerical_part * 60000)
-            elif unit_part == "s":
-                time_in_ms += int(numerical_part * 1000)
-            else:  # unit_part == "ms":
-                time_in_ms += int(numerical_part)
-            prev_end = unit_match.end()
-        last_part = time_string[prev_end:].strip()
-        if last_part:
-            time_in_ms += int(float(last_part) * 1000)
-        return time_in_ms
-    except ValueError as e:
-        # e might have been raised by any of the float() constructor
-        raise ValueError(
-            f'cannot parse "{time_string}" as a valid time in h/m/s/ms'
-        ) from e
-
-
-# todo: synchronize with web-component readme
-# https://github.com/ReadAlongs/Web-Component/blob/main/packages/studio-web/src/app/demo/demo.component.ts#L49
-# https://github.com/ReadAlongs/Web-Component/blob/main/packages/studio-web/src/app/demo/demo.component.ts#L328
-TEMPLATE_README_TXT = """
-Web Deployment Guide
-
-This folder has everything you need to host your ReadAlong on your own server.
-
-Your audio ({audio}), (optional) image(s) asset(s), and alignment ({text}) are stored in this folder.
-
-Your index.html file demonstrates the snippet and imports needed to host the ReadAlong on your server.
-
-Please host all assets on your server, include the font and package imports defined in the index.html in your website's imports, and include the corresponding <read-along> snippet everywhere you would like your ReadAlong to be displayed.
-
-        `
-
-WordPress Deployment Guide
-
-
-Setup the plugin (do this once)
-
-Install and activate our plugin 'wp-read-along-web-app-loader' on your WordPress site.
-
-
-Deploy the read-along
-
-Upload the {text} and {audio} to your Media Library of your WordPress site.
-
-Use the text editor to paste the snippet below in your WordPress page
-
-Replace assets/ with the path from your Media Library
-
-        ---- WP Deployment SNIPPET ----
-
-<!-- Here is how you declare the Web Component. Supported languages: en, fr -->
-[wp_read_along_web_app_loader version="{version}"]
-    <read-along href="{text}" audio="{audio}" theme="{theme}" language="en">
-        <span slot='read-along-header'>{header}</span>
-        <span slot='read-along-subheader'>{subheader}</span>
-    </read-along>
-[/wp_read_along_web_app_loader]
-
-        ----- END OF SNIPPET----
-
-"""
-
-
-def save_readme_txt(
-    output_path,
-    tokenized_xml_basename,
-    audio_basename,
-    header,
-    subheader,
-    theme,
-):
-    with open(output_path, "w", encoding="utf-8") as fout:
-        fout.write(
-            TEMPLATE_README_TXT.format(
-                version=CURRENT_WEB_APP_VERSION,
-                text=tokenized_xml_basename,
-                audio=audio_basename,
-                theme=theme,
-                header=header,
-                subheader=subheader,
-            )
-        )
+###########################################

```

Comment:
This PR changes the newline style on this file from LF to CRLF. Please revert that change, and maybe configure your editor to always use LF, and maybe even use `git config --global core.autocrlf input` unless you have other projects that need CRLF line ends.